docs/workers.md

```diff
@@ -502,11 +504,32 @@ after setting this option in the shared configuration!
 This style of configuration supersedes the legacy `synapse.app.appservice`
 worker application type.
 
+#### Push Notifications
+
+You can designate one generic worker to sending push notifications to
+[sygnal](https://github.com/matrix-org/sygnal) and email.
+This will stop the main synapse sending push notifications.
+Doesn't handle any REST endpoints itself, but you should specify its name in the
+[shared configuration](usage/configuration/config_documentation.md#pusher_instances)
+as follows:
```

Comment:
Not sure what you're driving at with the REST endpoints here. Is the point that you don't need to adjust reverse proxy config when configuring pusher workers? If so, how about something like

```
The workers responsible for sending push notifications can be defined using the `pusher_instances` option. For example:

...

Workers don't need to accept incoming HTTP requests to send push notifications, so no additional reverse proxy configuration is required for pusher workers.
```

Reply:
The information that a worker does not accept HTTP requests is available for different workers, therefore also here.